src/pipecat/processors/aggregators/llm_response.py

```diff
@@ -121,14 +108,12 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
     async def _push_aggregation(self):
         if len(self._aggregation) > 0:
             self._messages.append({"role": self._role, "content": self._aggregation})
-
-            # Reset the aggregation. Reset it before pushing it down, otherwise
-            # if the tasks gets cancelled we won't be able to clear things up.
-            self._aggregation = ""
-
```

Comment:
Watch out removing all this or interruptions will be broken. Mmmmm...just wondering that you probably didn't rebase?